src/components/MenuItem.js

```diff
@@ -22,24 +22,29 @@ const propTypes = {
     /** Text to display for the item */
     title: PropTypes.string.isRequired,
 
-    /** Boolean whether to display the ArrowRight icon */
-    shouldShowRightArrow: PropTypes.bool,
+    /** Boolean whether to display the right icon */
+    shouldShowRightIcon: PropTypes.bool,
 
     /** A boolean flag that gives the icon a green fill if true */
     success: PropTypes.bool,
+
+    // Overrides the icon for shouldShowRightIcon
+    iconRight: PropTypes.element,
```

Comment:
I'm getting a propType warning from this... been trying to track it down but not having any luck. Maybe you know?

![image](https://user-images.githubusercontent.com/1228807/118880905-a8759d80-b8af-11eb-9b07-c312ac05a49a.png)

Reply:
I think this should be changed 
```diff
- iconRight: PropTypes.element
+ iconRight: PropTypes.elementType
```

Reply:
@pranshuchittora would you mind holding on until @tgolen is able to reply, then create a new PR with these suggestions? Thanks!